src/components/form/GroupElement.tsx

```diff
@@ -58,47 +104,27 @@ export default class GroupElement extends React.Component<GroupElementProps, Gro
     public validate(): boolean {
         const errors: string[] = [];
 
-        if (this.props.required) {
-            if (this.state.groups.length < 1) {
-                errors.push(`${this.props.name} is required`);
-            }
+        if (this.props.required && !this.state.groups.length) {
+            errors.push(`${this.props.name} is required`);
         }
 
         this.setState({ errors });
 
         return errors.length === 0;
     }
 
-    private isValidNewOption({ label }: { label: string }): boolean {
-        if (!label) {
-            return false;
-        }
-
-        const lowered = label.toLowerCase();
-
-        return lowered.length > 0 && lowered.length <= 36 && /^[a-z0-9-][a-z0-9- ]*$/.test(lowered);
-    }
-
-    private createNewOption(arg: { label: string }): SearchResult {
-        const newOption = {
-            id: generateUUID(),
-            name: arg.label,
-            extraResult: true
-        } as SearchResult;
-
-        return newOption;
-    }
-
     public render(): JSX.Element {
         const createOptions: any = {};
 
         if (this.props.add) {
-            createOptions.isValidNewOption = this.isValidNewOption;
-            createOptions.createNewOption = this.createNewOption;
-            createOptions.createPrompt = 'New group: ';
+            createOptions.isValidNewOption = isValidNewOption;
+            createOptions.createNewOption = createNewOption;
+            createOptions.createPrompt = GROUP_PROMPT;
         }
 
-        const className: string = getSelectClass(this.state.errors.length);
+        const className = getSelectClass(this.state.errors.length);
+        const placeholder = this.props.placeholder || GROUP_PLACEHOLDER;
```

Comment:
`GroupElement` will use these values unless passed props. Can also be done w/ Reacts `defaultProps` class property.